applications/nrf5340_audio/src/audio/sw_codec_select.h

```diff
@@ -9,7 +9,8 @@
 
 #include <zephyr/kernel.h>
 #include <zephyr/net_buf.h>
-#include "channel_assignment.h"
+#include <zephyr/bluetooth/audio/audio.h>
```

Comment:
This is already included in "device_location.h", so no need to include it here as well

Reply:
Well, if one could imagine using this module without device_location.h, it should be here.